spark/src/main/scala/ai/chronon/spark/Driver.scala

```diff
@@ -732,8 +732,16 @@ object Driver {
     }
 
     def run(args: Args): Unit = {
-      val putRequest = args.metaDataStore.putConf(args.confPath())
-      val res = Await.result(putRequest, 1.hour)
+      val acceptedEndPoints = List("ZIPLINE_METADATA", "ZIPLINE_METADATA_BY_TEAM")
```

Comment:
We probably can make this a passed-in config, WDYT? @nikhilsimha

Reply:
agreed - we could take this from conf